packages/compose/src/hooks/use-resize-observer/index.native.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Internal dependencies
+ */
+import useResizeObserver from './legacy/index.native';
+
+export default useResizeObserver;
```

Comment:
Could be just one line: `export { default as useResizeObserver } from './legacy';`

Reply:
True, will do that and merge.